pkg/kubelet/dockertools/manager.go

```diff
@@ -263,6 +265,37 @@ func (sc *reasonInfoCache) Get(uid types.UID, name string) (reasonInfo, bool) {
 	}
 }
 
+func (dm *DockerManager) parallelPullSupportExists(serializeImagePulls util.BoolFlag) bool {
```

Comment:
Thats fine. I am just wondering if we could choose some other name because having a function for parallel pull and then passing a variable named serialImagePulls sounds odd from readability point of view.